### PR TITLE
Streamline clipboard.js usage in docs, and fetch it from cdn

### PR DESCRIPTION
**Pull request by GeoSot**

### CODE REVIEW

File: site/layouts/partials/scripts.html [hunk 1]
```diff
@@ -4,15 +4,16 @@
   <script src="/docs/{{ .Site.Params.docs_version }}/dist/js/bootstrap.bundle.js"></script>
 {{- end }}
 
+<script src="https://cdn.jsdelivr.net/npm/clipboard@latest/dist/clipboard.min.js"></script>
```
Comment (GeoSot) [inline site/layouts/partials/scripts.html:7]:
```suggestion
<script src="https://cdn.jsdelivr.net/npm/clipboard@2.0.11/dist/clipboard.min.js"></script>
```

Not sure if is better to use a specific version

Comment (mdo) [reply site/layouts/partials/scripts.html:7]: I think it's best to use a specific version for security reasons, yeah?

Comment (GeoSot) [reply site/layouts/partials/scripts.html]: Done ;)

**Review by XhmikosR — CHANGES_REQUESTED** [review]
We should not use the CDN

**Comment by GeoSot** [comment]
Can you explain the reason ? We use docsearch and stackblitz through cdn. What is the difference on this?

**Comment by XhmikosR** [comment]
None of those should be fetched from the CDN either. https://github.com/twbs/bootstrap/issues/27412

**Comment by GeoSot** [comment]
So maybe a package.json requirement would be better than the existing implementation too ( Hugo can handle imports)

**Comment by XhmikosR** [comment]
I'm aware it does. My problem is that at the time when I tested it, Hugo didn't throw an error if a mount is not found. That's why I haven't implemented this.

**Comment by GeoSot** [comment]
You may rebase the branch you have tried to, and if you like to forward it to me, to have a look to.

PS: Till then, I suppose it is better to have clipboard.js imported the same way as the other scripts, just for consistency

**Comment by GeoSot** [comment]
Did a try [here](https://github.com/twbs/bootstrap/tree/gs/make-docs-js-build). No need for mounts 😕

**Pull request closed without merging**